common/thorhelper/thorsoapcall.cpp

```diff
@@ -254,6 +254,24 @@ class Url : public CInterface, implements IInterface
     }
 };
 
+//If the above Url class was centrally located, we could define this in jtrace instead.
+//http span standards documented here: https://opentelemetry.io/docs/specs/semconv/http/http-spans/
+void setSpanURLAttributes(ISpan * clientSpan, const Url & url)
+{
+    if (clientSpan == nullptr)
+        return;
+
+    clientSpan->setSpanAttribute("http.request.method", "POST"); //apparently hardcoded to post
```

Comment:
http.request.method is required for http spans per otel standards, but if the action is http call, the service name is used in the actual request, making unclear how we should handle this.